lib/cfme/migration_analytics/engine.rb

```diff
@@ -4,7 +4,7 @@ class Engine < ::Rails::Engine
       isolate_namespace Cfme::MigrationAnalytics
 
       def self.vmdb_plugin?
-        false # TODO: this should be changed back to true when we re-enable the menu entry below
+        true
```

Comment:
Totally untested, but could this just be `def self.vmdb_plugin?; Settings.prototype.migration_analytics.enabled; end` ?

Reply:
I think the problem with that is that when the appliance is built, if this is false it won't build the UI assets for the plugin, so changing the settings flag and rebooting the appliance wouldn't be enough to enable it. That's the problem @bthurber was running into this morning when he tried to enable it by hand. I think we need this to be true so the assets get built regardless of the setting.

Reply:
Right, vmdb_plugin drives both runtime *and* build time things.  In this case the UI assets were not available when set to false.